docs/source/getting-started/keras/index.rst

```diff
@@ -0,0 +1,8 @@
+Getting Started with MLflow + Keras 3.0
```

Comment:
Can we add a landing page for this that is similar to the one created for transformers in this PR? https://github.com/mlflow/mlflow/pull/10360